### PR TITLE
MInor spacing fixes in src/simple.jl

### PR DESCRIPTION
**Pull request by miguelraz**
[ci skip]

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
Thanks! Will merge when CI finishes.

**Comment by ChrisRackauckas** [comment]
AppVeyor failed. Something about the common.jl solve dispatch no longer works.

File: src/common.jl [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ## Common Interface Solve Functions
 
-function solve{uType,tType,isinplace,F,Method,LinearSolver}(
-    prob::AbstractODEProblem{uType,tType,isinplace,F},
+function solve{uType, tType, inplace, F, Method, LinearSolver}(
```
Comment (ChrisRackauckas) [inline src/common.jl]: isinplace

**Comment by miguelraz** [comment]
Got it! Darn insert key...

**Pull request merged**